index.js

```diff
@@ -35,12 +36,44 @@ function getConfigs() {
     skipTags: devEnv
       ? yn(process.env.SKIP_TAGS)
       : yn(core.getInput("skip-tags")),
+    retry: true,
   };
 }
 
+const ThrottledOctokit = Octokit.plugin(throttling);
+
 async function run() {
   const configs = getConfigs();
-  const octokit = new Octokit();
+  const octokit = new ThrottledOctokit({
+    throttle: {
+      onRateLimit: (retryAfter, options) => {
+        console.error(
+          `Request quota exhausted for request ${options.method} ${options.url}, remaining requests: ${options.request.retryCount}`
```

Comment:
Isn't that the number of retries already done?

Reply:
Ahh, you're right, thanks.

Reply:
5369a8b
38ad24d